lavamoat/browserify/beta/policy.json

```diff
@@ -5664,6 +5799,32 @@
         "react-markdown>vfile>vfile-message": true
       }
     },
+    "viem": {
+      "globals": {
+        "AbortController": true,
+        "Image": true,
+        "Request": true,
+        "TextDecoder": true,
+        "TextEncoder": true,
+        "atob": true,
+        "btoa": true,
+        "clearInterval": true,
+        "clearTimeout": true,
+        "console.error": true,
+        "fetch": true,
```

Comment:
What is being fetched?

Reply:
nothing from our end... this was automatically generated via `yarn lavamoat:auto`. Can we manually remove this? What is the process?